core/collector.class.inc.php

```diff
@@ -716,32 +716,55 @@ public function Synchronize($iMaxChunkSize = 0)
 		$sResult = self::CallItopViaHttp("/synchro/synchro_exec.php?login_mode=$sLoginform",
 			$aData);
 
-		$iErrorsCount = 0;
+		$iErrorsCount = $this->ParseSynchroExecOutput($sResult);
+
+		return ($iErrorsCount == 0);
+	}
+
+	/**
+	 * @since N°6771
+	 * Detects synchro exec errors and finds out details message
+	 * @param string $sResult: synchro_exec.php output
+	 *
+	 * @return int: error count
+	 * @throws \Exception
+	 */
```

Comment:
```suggestion
	/**
	 * Detects synchro exec errors and finds out details message
	 * @param string $sResult synchro_exec.php output
	 * @return int error count
	 * @throws Exception
	 * @since 1.3.1 N°6771
	 */
```